platform/mbed_retarget.cpp

```diff
@@ -1053,12 +1053,13 @@ extern "C" uint32_t  __HeapLimit;
 extern "C" int errno;
 
 // Dynamic memory allocation related syscall.
-#if defined(TARGET_NUVOTON)
+#if (defined(TARGET_NUVOTON) || defined(TWO_RAM_REGIONS))
```

Comment:
cleaner would be nuvoton would also define two ram regions so we would have just one config and no target here

Reply:
0xc0170 you are right.
I suggest we create an issue in mbed-os so that NUVOTON can use the same define once this PR is merged. What do you think ?

Reply:
Yes, can be done separately, would be nice o t have a tracking issue so we do not forget, thanks